filament/backend/src/metal/MetalHandles.h

```diff
@@ -164,6 +163,19 @@ struct MetalProgram : public HwProgram {
     bool isValid = false;
 };
 
+struct PixelBufferShape {
+    size_t bytesPerPixel;
+    size_t bytesPerRow;
+    size_t bytesPerSlice;
+    size_t totalBytes;
```

Comment:
do all these need to be `size_t` as opposed to `uint32_t`?

Reply:
`uint32_t` should be fine.